modules/manage/pages/manage-expiration.adoc

```diff
@@ -171,31 +170,21 @@ Setting `maxTTL` to a non-zero value has two effects:
 * You cannot explicitly set a document's expiration to be longer than the `maxTTL` setting. 
 If you try to set the expiration to a value larger than the collection or bucket's `maxTTL` setting, Couchbase Server uses the `maxTTL` value instead.
 
-You can set the `maxTTL` setting for buckets and collections when you initially create them. You can change the `maxTTL` setting of a bucket after you create it.
-
 === Set maxTTL Using Couchbase Server Web Console
 
 When you create a new bucket or collection using the Web Console, you can enable the `maxTTL` setting.
 
-To enable automatically expiring documents when creating a bucket using the Web Console:
+To change a bucekt's `maxTTL` setting when creating or editing it:
 
-. In the *Add Data Bucket* dialog, expand the *Advanced bucket settings* section.
+. When edting or adding a bucket, expand the *Advanced bucket settings* section.
 . Under *Bucket Max Time-To-Live* select *Enable*.
-. In the text field, enter the default TTL for documents in seconds.
+. Under *Bucket Max Time-To-Live*, select or clear the *Enable* box to enable or turn off automatic expiration.
+, If you're enabling automatic expiration, enter the number of seconds the documents should exist before expiring in the text field.
 
 To enable expiration when creating a collection using the Web Console, enter a non-zero value in the *Add Collection to _scope name_ Scope* dialog's *Collection Max Time-To-Live* field.
 
-You can also use the Web Console to change the `maxTTL` setting of an existing bucket:
-
-. Click btn:[Buckets].
-. Click the name of the bucket whose `maxTTL` setting you want to change.
-. Click btn:[Edit].
-. Expand the *Advanced bucket settings* section.
-. Under *Bucket Max Time-To-Live*, select or clear the *Enable* box to enable or turn off automatic expiration.
-, If you're enabling automatic expiration, enter the number of seconds the documents should exist before expiring in the text field.
-. Click btn:[Save Changes].
+Once created, you can edit the `maxTTL` setting by clicking *Edit TTL* in the list of collections on the *Scopes & Collections* page.
 
-You cannot change the `maxTTL` of a collection after you create it.
  
 === Set maxTTL Using the REST API
```

Comment:
I don't know how to add a comment to a line that hasn't changed. Down below the changing of maxTTL is using "-X POST".  It cannot be changed via POST.  Instead it should be "-X PATCH" .